packages/TelegramClient-Core.package/TCCAuthHandler.class/instance/setTDLibParams.st

```diff
@@ -12,6 +12,6 @@ setTDLibParams
 			'api_hash' -> self apiHash.
 			'system_language_code' -> 'en'.
 			'device_model' -> 'Desktop'.
-			'system_version' -> self client specificClient type.
+			'system_version' -> self core client specificClient type.
```

Comment:
Does this hurt the law of Demeter? 
All of the other code looks fine :)

Reply:
The answer is **yes**, this hurts Law of Demeter significantly. The fact that we are now calling `self core client` instead of `self client` doesn't change anything though, `self client` returned `self core client`

Reply:
Unfortunately, other than creating type on TeleMock and on Core and then propagating the answer through, we didn't see any other way to implement. Given that his only happens once, we thought it would be better to have this line once than introduce 2 new methods for it

Reply:
I thinks its fine to hurt LoD in this case

Reply:
Alright, thank you for the clarification :)